deepmd/utils/data.py

```
@@ -583,13 +594,16 @@ def _get_nframes(self, set_name: DPPath) -> int:
-        else:
-            coord = path.load_numpy().astype(GLOBAL_NP_FLOAT_PRECISION)
-        if coord.ndim == 1:
-            coord = coord.reshape([1, -1])
-        nframes = coord.shape[0]
+        # Read only the header to get shape
+        with open(str(path), "rb") as f:
+            version = np.lib.format.read_magic(f)
+            if version[0] == 1:
+                shape, fortran_order, dtype = np.lib.format.read_array_header_1_0(f)
+            elif version[0] in [2, 3]:
+                shape, fortran_order, dtype = np.lib.format.read_array_header_2_0(f)
```

Ruff (0.14.4)

Unpacked variable `fortran_order` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)